tests/saver/test_save_prediction.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import pandas as pd
+
+from scipy.stats import norm
+
+from cascade.saver.save_prediction import uncertainty_from_prediction_draws, _predicted_to_uploadable_format
+
+
+def test_uncertainty_from_prediction_draws():
+    rng = np.random.RandomState(4398221)
+
+    num_draws = 100
+    locations = [1, 2]
+    age_lower, age_upper = zip(*[(0, 0.5), (50, 60)])
+    sexes = [-0.5]
+    time_lower, time_upper = zip(*[(y, y) for y in [2000, 2020]])
+    integrands = ["prevalence", "Sincidence"]
+
+    df = pd.DataFrame(
+        index=pd.MultiIndex.from_product(
+            [locations, age_lower, age_upper, time_lower, time_upper, integrands, sexes],
+            names=["location", "age_lower", "age_upper", "time_lower", "time_upper", "integrand", "s_sex"],
```

Comment:
Man, these multiindex are so complicated.

Reply:
If you have a cleaner way of constructing this dataframe I'm all ears. I could do it as a big loop but I think that's actually more distracting from the test and this trick (and it's totally a dumb trick) is the canonical way of doing this as far as I know.

Reply:
My method for Pandas is to review the docs on the data structure, make a stab, and test it. Anything that passes tests is going to have to be good enough.